BulletinBoard.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "BulletinBoard"
-  s.version      = "2.0.1"
+  s.version      = "3.0.0"
```

Comment:
Can you add `s.swift_version = "4.2"`? This was added so CP can officially set the correct `SWIFT_VERSION` for the target

Reply:
Added it, thanks for the heads-up 👍